### ISSUE TITLE
Failed to get API group resources in 0.23.0

### ISSUE THREAD

**Issue by gabriel-suela** [Altinity/clickhouse-operator#1340]
im getting this error in clickhouse-operator after i upgrade the chart version from 0.22.2 to 0.23.0. Already tried 0.23.1 and got the same error

```
2024-02-09T20:26:05Z    ERROR    controller-runtime.source.EventHandler    failed to get informer from cache    {"error": "failed to get API group resources: unable to retrieve the complete list of server AP │
│ Is: clickhouse-keeper.altinity.com/v1: the server could not find the requested resource"}                                                                                                                       │
│ sigs.k8s.io/controller-runtime/pkg/internal/source.(*Kind).Start.func1.1                                                                                                                                        │
│     /clickhouse-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/source/kind.go:68                                                                                                                   │
│ k8s.io/apimachinery/pkg/util/wait.loopConditionUntilContext.func1                                                                                                                                               │
│     /clickhouse-operator/vendor/k8s.io/apimachinery/pkg/util/wait/loop.go:62                                                                                                                                    │
│ k8s.io/apimachinery/pkg/util/wait.loopConditionUntilContext                                                                                                                                                     │
│     /clickhouse-operator/vendor/k8s.io/apimachinery/pkg/util/wait/loop.go:63                                                                                                                                    │
│ k8s.io/apimachinery/pkg/util/wait.PollUntilContextCancel                                                                                                                                                        │
│     /clickhouse-operator/vendor/k8s.io/apimachinery/pkg/util/wait/poll.go:33                                                                                                                                    │
│ sigs.k8s.io/controller-runtime/pkg/internal/source.(*Kind).Start.func1                                                                                                                                          │
│     /clickhouse-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/source/kind.go:56 
```

**Comment by Slach** [comment]
do you use helm chart?
did you upgrade CRDS separately before upgrade release?

**Comment by alex-zaitsev** [comment]
@gabriel-suela , the error message indicates that new CRD for Keeper is missing in your Kuberentes. It may happen if you upgraded it via Helm. That should be fixed in 0.23.1, could you try to upgrade one more time? On the other hand, Helm can not upgrade CRDs properly, so re-install of operator is probably better.

**Comment by gabriel-suela** [comment]
@Slach @alex-zaitsev  managed to make it work by applying the CRD manually. Ty for the responses guys.
